lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -12,12 +12,16 @@
         {% endif %}
     </span>
 {% endblock %}
-{% macro settings_checkbox(label, setting, sub_setting, field_name, default=False) %}
+{# Tri-state checkbox with enabled/disabled/use-default options. #}
+{% macro settings_checkbox(label, setting, sub_setting, field_name, default=None) %}
     {% call macros.field_body(label) %}
-        <label class="checkbox">
-            <input {% if instance.settings.get(setting, sub_setting, default) %}checked{% endif %}
-                   type="checkbox"
-                   name="{{ setting }}.{{ sub_setting }}">
+        {% set value = instance.settings.get(setting, sub_setting, default) %}
+        <label class="select">
+            <select name="{{setting}}.{{sub_setting}}">
+              <option value="true" {% if value is true %}selected{% endif %}>Enabled</option>
+              <option value="false" {% if value is false %}selected{% endif %}>Disabled</option>
+              <option value="none" {% if value is none %}selected {% endif %}>Default</option>
```

Comment:
Perhaps the `Default` option could dynamically show what is currently the default: `Default (enabled)` or `Default (disabled)`.

However, I'm not sure if that's easy to infer.

Reply:
It would be nice if this could be indicated. Ideally we'd want the default value passed through from the source of truth on the Python side so that the value isn't duplicated. I didn't know off-hand how best to do this.

Reply:
There's a default parameter here to do that, ideally that should come from the backend and not the template but for now rendering the "Default" label using the `default` parameter here should should be enough.